samples/widgets/button.cpp

```diff
@@ -618,17 +631,18 @@ void ButtonWidgetsPage::OnButton(wxCommandEvent& WXUNUSED(event))
 // bitmap button stuff
 // ----------------------------------------------------------------------------
 
-wxBitmap ButtonWidgetsPage::CreateBitmap(const wxString& label)
+wxBitmap ButtonWidgetsPage::CreateBitmap(const wxString& label, const wxArtID type)
 {
-    wxBitmap bmp(180, 70); // shouldn't hardcode but it's simpler like this
+    wxBitmap bmp(FromDIP(180), FromDIP(70)); // shouldn't hardcode but it's simpler like this
```

Comment:
Very minor, but I think this might be slightly better, perhaps?
```suggestion
    wxBitmap bmp(FromDIP(wxSize(180, 70)); // shouldn't hardcode but it's simpler like this
```

Reply:
Yes, but with matching brackets 😉.